iceoryx_posh/include/iceoryx_posh/popo/publisher_options.hpp

```diff
@@ -39,6 +42,12 @@ struct PublisherOptions
 
     /// @brief The option whether the publisher should block when the subscriber queue is full
     SubscriberTooSlowPolicy subscriberTooSlowPolicy{SubscriberTooSlowPolicy::DISCARD_OLDEST_DATA};
+
+    /// @brief serialization of the PublisherOptions
```

Comment:
Unfortunately, you have to update the copyright header...

Reply:
Done in all files